@xen-orchestra/web-core/lib/components/ui/table-actions/UiTableActions.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="ui-table-actions">
+    <UiActionsTitle>
+      <slot />
+    </UiActionsTitle>
+    <slot name="actions" />
+  </div>
```

Comment:
From what I see in the design system, maybe it could be useful to have another (optional) slot that can hold the `GroupedBy` component (which is not done yet). What do you think?